### PR TITLE
Update to egui@0.28

### PR DESCRIPTION
**Pull request by v0x0g**
Follows https://github.com/emilk/egui/releases/tag/0.28.0

### CODE REVIEW

**Comment by v0x0g** [comment]
Upon further testing I realize this is currently incomplete, and does not build, due to changes with the removal of `egui::Event::Scroll`.

I will attempt to make the appropriate fixes and update my PR shortly.

**Comment by emilk** [comment]
See the migration guide for help: https://github.com/emilk/egui/blob/master/CHANGELOG.md#-migration

**Comment by v0x0g** [comment]
@emilk Fixed and updated, everything seems to be working fine. I have just tested:
- My own app `rayna`
- The `egui-demo`
- `./check.sh`

I bumped the version to `0.14.1`, as none of the modifications impact the public API at all.

**Review by emilk — APPROVED** [review]

File: src/lib.rs [hunk 1]
```diff
@@ -259,14 +259,13 @@ impl EguiMq {
...
-        let event = if self.egui_input.modifiers.ctrl {
-            // Treat as zoom instead:
-            egui::Event::Zoom((delta.y / 200.0).exp())
-        } else {
-            egui::Event::Scroll(delta)
-        };
-        self.egui_input.events.push(event);
+        self.egui_input.events.push(egui::Event::MouseWheel {
+            modifiers,
+            unit: egui::MouseWheelUnit::Line,
```
Comment (emilk) [inline src/lib.rs:266]: Is this really then unit miniquad uses? Sadly it is not documented: https://docs.rs/miniquad/latest/src/miniquad/event.rs.html#170

Maybe @not-fl3 knows?

**Review by emilk — APPROVED** [review]

File: src/lib.rs (hunk 1 above)
Comment (v0x0g) [reply src/lib.rs:266]: Yes, I also noticed that it's not documented there.

I ended up doing some testing and adding `dbg!(delta)` inside the event handler, which ended up printing multiples of one every time I scrolled:  `[1.0, 0.0], [-1.0, 0.0], [0.0, 1.0]` (every single event was `+- 1.0`).  I tested using both my mouse and trackpad, and got a consistent `[1.0, 0.0]` for every  'click' of the scroll wheel, so I'm, fairly confident it's in lines.

Comment (not-fl3) [reply src/lib.rs:266]: I don't :| 
On most platforms its just a raw data from an OS message, on some its fixed 1.0/-1.0.
We do not normalize it in any way.

**Comment by emilk** [comment]
Published as 0.15.0

**Pull request merged**